packages/forms-web-app/__tests__/unit/controllers/before-you-start/use-existing-service-application-type.test.js

```diff
@@ -17,7 +17,7 @@ describe('controllers/full-appeal/use-existing-service-application-type', () =>
 	it('Test getUseExistingServiceApplicationType method calls the correct template', async () => {
 		await getUseExistingServiceApplicationType(req, res);
 
-		expect(res.render).toBeCalledWith(USE_EXISTING_SERVICE_APPLICATION_TYPE, {
+		expect(res.render).toHaveBeenCalledWith(USE_EXISTING_SERVICE_APPLICATION_TYPE, {
```

Comment:
are these distict

Reply:
toBeCalledWith appears to be marked as deprecated